### DISCUSSION TITLE
只有pro专属功能，没有AI翻译的会员选择

### DISCUSSION THREAD

**Discussion by pengzy-cosmo** [immersive-translate/immersive-translate#2374] [Ideas]
### 新功能 | New Feature

希望有一种较低价格的会员选项，只享受pro专属功能，不提供AI翻译

### 为什么您想要添加这些功能 | Why you want to add these features

很多人自己有相关AI服务的API，只想用插件中的Pro专属功能，不需要额外的AI翻译

**Comment by theowenyoung** [comment]
https://immersivetranslate.com/accounts/pre-order?price_id=price_1PWg9EGc8iUjvqOFATBhujeS

纯功能会员，不包含任何额度。

LeGibet ***@***.***>于2024年12月18日 周三22:08写道：
